reconcile/gitlab_housekeeping.py

```diff
@@ -19,6 +19,26 @@
 
 QONTRACT_INTEGRATION = 'gitlab-housekeeping'
 
+def clean_pipelines(dry_run, gl, pipeline_timeout=60):
+    DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+    now = datetime.utcnow()
+
+    pipelines = gl.get_pipelines(status='pending')
+
+    for p in pipelines:
+        last_update_date = datetime.strptime(p.updated_at, DATE_FORMAT)
+
+        pipeline_elapsed_time = now.date() - last_update_date.date()
```

Comment:
Don't change it, but for the future, consider whether for this short-lived object `elapsed` wouldn't be simpler to understand.

Reply:
But do try to change it to seconds and integer operations. It'll be less code and it will also be faster.